tests/db.spec.js

```diff
@@ -1,12 +1,38 @@
 require('dotenv').config();
 
-const { rebuildDb } = require('../db/init_db')
+const { buildDb } = require('../db/init_db')
 
 describe('Database', () => {
     beforeAll(async() => {
-      await rebuildDB();
+      await buildDB();
```

Comment:
I think this needs to be either DB or db

`const { buildDb } = require('../db/init_db') <--- the casing is not the same

describe('Database', () => {
    beforeAll(async() => {
      await buildDB();` <--- the casing is not the same